src/checkers/inference/InferenceMain.java

```diff
@@ -290,9 +287,9 @@ private InferrableChecker getRealChecker() {
                         InferenceOptions.checker, true, ClassLoader.getSystemClassLoader()).newInstance();
                 realChecker.init(inferenceChecker.getProcessingEnvironment());
                 realChecker.initChecker();
-                logger.finer(String.format("Created real checker: %s", realChecker));
+                logger.info(String.format("Created real checker: %s", realChecker));
```

Comment:
Based on the log4j official site: `FINER` in JUL are equal to `TRACE` in log4j. Can we keep the original log level?

here is the convert log level table from JUL to log4j:

https://logging.apache.org/log4j/2.x/log4j-jul/